### ISSUE TITLE
The problem of "targets" parameter

### ISSUE THREAD

**Issue by LeiShe1** [voletiv/mcvd-pytorch#21]
Thanks to the author's work, the experiment is very good!
I want to ask about a parameter problem in the code:
![image](https://github.com/voletiv/mcvd-pytorch/assets/42711890/e6898737-9b51-4edc-9d62-8456eb54c596)
This is the targets parameter of stochastic_moving_mnist.
![image](https://github.com/voletiv/mcvd-pytorch/assets/42711890/c3a5a7c4-0579-4d26-aecb-52b572b7eeb6)
This is the targets parameter of cityscapes.
The targets parameter seems to return different values in different code. What is the function of this parameter? I want to use my own data set. How should this parameter be set.
Looking forward to your reply, thank you!

**Comment by AlexiaJM** [comment]
Hi @LeiShe1, the targets represent categorical data (label of imagenet, identity of a person on CelebA). In the end we never use targets in the code (see for yourself with a ctrl-f through the codebase), so just put a random value or torch.tensor(1) in its place and ignore it.
